x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/space_aware_privilege_section/space_aware_privilege_section.tsx

```diff
@@ -194,20 +194,21 @@ export class SpaceAwarePrivilegeSection extends Component<Props, State> {
       return null;
     }
 
-    const addPrivilegeButton = (
-      <EuiButton
-        color="primary"
-        onClick={this.addSpacePrivilege}
-        iconType={'plusInCircle'}
-        data-test-subj={'addSpacePrivilegeButton'}
-        isDisabled={!hasAvailableSpaces || !this.props.editable}
-      >
-        <FormattedMessage
-          id="xpack.security.management.editRole.spacePrivilegeSection.addSpacePrivilegeButton"
-          defaultMessage="Add Kibana privilege"
-        />
-      </EuiButton>
-    );
+    const addPrivilegeButton =
+      !hasAvailableSpaces || !this.props.editable ? null : (
+        <EuiButton
+          color="primary"
+          onClick={this.addSpacePrivilege}
+          iconType={'plusInCircle'}
+          data-test-subj={'addSpacePrivilegeButton'}
+          isDisabled={!hasAvailableSpaces || !this.props.editable}
```

Comment:
Very minor nit: Should we not bother to set isDisabled here? Looks like we're not even going to render the button if there are no spaces or it's not editable. It doesn't really matter wither way.

Reply:
Yeh, you're right this would never be reached. I don't think it hurts so will leave as is for now.